### PR TITLE
JDBC: first three task from jdbc

### PR DESCRIPTION
**Pull request by kristiyanpetkov**
This PR addresses first three tasks from JDBC. Here is a short description for each of them: 
 * First task is using JDBC for communication with the database and execution of different type of queries  (select, update, delete)
 * Second task  is using relations between tables. It uses People and Trip as example. 
 * Third task is created using triggers for adding of history support for a given data

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -0,0 +1 @@
+jdbc
```
Comment (EmilGeorgiev) [inline .idea/.name]: all .idea/ directory should be added in .gitignore file

Comment (kristiyanpetkov) [inline .idea/.name]: fixed

**Comment by kristiyanpetkov** [comment]
@EmilGeorgiev , i fixed the first task and added tests.

File: src/com/clouway/firsttask/Person.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
+package com.clouway.firsttask;
+
+/**
+ * Created by clouway on 16-3-1.
+ */
+public class Person {
+    private String egn;
+    private String name;
+    private int age;
+    private String gender;
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/Person.java]: This class should be Immutable. An object is considered immutable if its state cannot change after it is constructed.
https://docs.oracle.com/javase/tutorial/essential/concurrency/immutable.html

To make this class immutable make all fields ```public final``` and remove all setter and getter methods

File: src/com/clouway/firsttask/PersonDao.java [hunk 3]
```diff
@@ -0,0 +1,234 @@
+package com.clouway.firsttask;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by clouway on 16-2-24.
+ */
+public class PersonDao {
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDao.java]: PersonDAO

File: src/com/clouway/firsttask/PersonDao.java [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+    private String DB_URL;
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDao.java]: by convention this name ```DB_URL``` is for constant values. Change to ```dbHost```, ```dbEndpoint``` or something else

File: src/com/clouway/firsttask/PersonDao.java [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+    }
+
+    public void connectToDatabase() {
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
+            connection = DriverManager.getConnection(DB_URL, user, password);
+        } catch (SQLException | ClassNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDao.java]: this method is unnecessary we will pass ```Connection``` as parameter of constructor.

File: src/com/clouway/firsttask/PersonDao.java [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    public List<Person> findAll() {
+        Statement stmt = null;
+        ResultSet resultSet = null;
+        List<Person> tableContain = new ArrayList<Person>();
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDao.java]: rename ```tableContain``` to ```persons```

File: src/com/clouway/firsttask/PersonDao.java [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+            connection = DriverManager.getConnection(DB_URL, user, password);
+        } catch (SQLException | ClassNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void register(Person person) {
+        PreparedStatement stmt = null;
+        try {
+            stmt = connection.prepareStatement("INSERT INTO peopleinfo (egn,name,age,gender) values (?,?,?,?)");
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDao.java]: i think people is enough for name of table.

File: src/com/clouway/firsttask/PersonDao.java [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+                if (stmt != null) {
+                    stmt.close();
+                }
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    public void delete(String egn) {
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDao.java]: rename ```delete``` to ```deleteByEGN```.  ```delete(String) ``` is not readable.

File: src/com/clouway/firsttask/PersonDao.java [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+            }
+        }
+        return person;
+    }
+
+
+    public List<Person> like(String columName, String wildcard) {
+        Statement stmt = null;
+        ResultSet resultSet = null;
+        List<Person> tableContain = new ArrayList<Person>();
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDao.java]: persons

File: src/com/clouway/firsttask/PersonDao.java [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+                    resultSet.close();
+                }
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+        return tableContain;
+    }
+
+    public void deleteTableContent() {
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDao.java]: deleteAll or dropTable

File: src/com/clouway/secondtask/core/Demo.java [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+import com.clouway.secondtask.dataaccesslayer.PeopleTripJdbcImpl;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+/**
+ * Created by clouway on 16-2-23.
+ */
+public class Demo {
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/core/Demo.java]: If we have tests this class is unnecessary

File: src/com/clouway/secondtask/core/Person.java [hunk 12]
```diff
@@ -0,0 +1,83 @@
+package com.clouway.secondtask.core;
+
+/**
+ * Created by clouway on 16-2-24.
+ */
+public class Person {
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/core/Person.java]: this class can extracted in common package and can be used from another packages. Now we have duplication of this class ```Person```

File: src/com/clouway/secondtask/core/PersonDao.java [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * Created by clouway on 16-2-24.
+ */
+public interface PersonDao {
+
+    void create(Person person);
+
+    void update(Person person);
+
+    List<Person> getPeople();
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/core/PersonDao.java]: findAll()

File: src/com/clouway/secondtask/core/PersonDao.java [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+
+    List<Person> getAllPeopleNameWhichStartWithSameCharacters(String phrase);
+
+    void deletePeople(String chooseTableOrContent);
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/core/PersonDao.java]: when in name of interface we have Person this word is not necessary to present and in method names (```getPeople()```, ```deletePeople()```) this is smell

File: src/com/clouway/secondtask/core/Trip.java [hunk 15]
```diff
@@ -0,0 +1,76 @@
+package com.clouway.secondtask.core;
+
+import java.util.Date;
+
+/**
+ * Created by clouway on 16-2-24.
+ */
+public class Trip {
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/core/Trip.java]: immutable

File: src/com/clouway/secondtask/core/TripDao.java [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+/**
+ * Created by clouway on 16-2-24.
+ */
+public interface TripDao {
+
+    void create(Trip trip);
+
+    void update(Trip trip);
+
+    List<Trip> getTrips();
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/core/TripDao.java]: .

File: src/com/clouway/secondtask/core/TripDao.java [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+
+    List<String> getMostVisitedCities();
+
+    void deleteTrip(String chooseTableOrContent);
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/core/TripDao.java]: .

File: src/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbcImpl.java [hunk 18]
```diff
@@ -0,0 +1,360 @@
...
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+        return citiesOrderedByMostVisited;
+    }
+
+    @Override
+    public List<Person> findAllPeopleInTheSameCityAtTheSameTime(java.util.Date arrival, java.util.Date departure) {
+        List<Person> peopleInSameCityAtTheSameTime = new ArrayList<Person>();
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbcImpl.java]: people

File: test/com/clouway/firsttask/Tests.java [hunk 19]
```diff
@@ -0,0 +1,94 @@
...
+    }
+
+    @After
+    public void disconnect() {
+        personDao.closeConnection();
+    }
+
+
+    @Test
+    public void registerPerson() {
```
Comment (EmilGeorgiev) [inline test/com/clouway/firsttask/Tests.java]: this test is unnecessary we use method register in another tests

File: test/com/clouway/firsttask/Tests.java [hunk 20]
```diff
@@ -0,0 +1,94 @@
...
+    }
+
+    @Test
+    public void findAll() {
+        Person myperson = new Person("9109232202", "Kristiyan Petkov", 24, "male");
+        personDao.register(myperson);
+        Person myperson2 = new Person("9209232201", "Ivan Ivanov", 23, "male");
+        personDao.register(myperson2);
+        List<Person> tableContain = personDao.findAll();
+        assertThat(tableContain.size(), is(2));
```
Comment (EmilGeorgiev) [inline test/com/clouway/firsttask/Tests.java]: instead expect size of list we should expect content of list. 
```javascript
  List<Person> got = personDao.findAll();
  List<Person> want = new ArrayList()
  want.add(myperson1);
  want.add(myperson2);
  assertThat(got, is(want))
```

File: test/com/clouway/firsttask/Tests.java [hunk 21]
```diff
@@ -0,0 +1,94 @@
...
+    @Test
+    public void like() {
+        Person myperson = new Person("9109232202", "Kristiyan Petkov", 24, "male");
+        personDao.register(myperson);
+        Person myperson2 = new Person("9109232203", "Krasimir Ivanov", 24, "male");
+        personDao.register(myperson2);
+        Person myperson3 = new Person("9109232204", "Stefan Petkov", 24, "male");
+        personDao.register(myperson3);
+        List<Person> likeWildcard = personDao.like("name", "Kr%");
+        assertThat(likeWildcard.size(), is(2));
```
Comment (EmilGeorgiev) [inline test/com/clouway/firsttask/Tests.java]: .

File: test/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbvImplTest.java [hunk 22]
```diff
@@ -0,0 +1,96 @@
...
+        peopleTripJdbc.create(new Person("Grigor Dimitrov", "8104221534", 25, "gdimitrov@abv.bg"));
+        peopleTripJdbc.update(myperson);
+        List<Person> actual = peopleTripJdbc.getPeople();
+        List<Person> expected = new ArrayList<Person>();
+        expected.add(myperson);
+        assertThat(actual, is(expected));
+    }
+
+    @Test
+    public void registerPerson() throws Exception {
```
Comment (EmilGeorgiev) [inline test/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbvImplTest.java]: unneccesary tests

File: test/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbvImplTest.java [hunk 23]
```diff
@@ -0,0 +1,96 @@
...
+        List<Person> actual = peopleTripJdbc.getPeople();
+        List<Person> expected = new ArrayList<Person>();
+        expected.add(myperson);
+        assertThat(actual, is(expected));
+
+
+    }
+
+    @Test
+    public void registerTrip() throws Exception {
```
Comment (EmilGeorgiev) [inline test/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbvImplTest.java]: .

File: src/com/clouway/firsttask/Person.java (hunk 2 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/Person.java]: done

File: src/com/clouway/firsttask/PersonDao.java (hunk 3 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDao.java]: done

File: src/com/clouway/firsttask/PersonDao.java (hunk 4 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDao.java]: done

File: src/com/clouway/firsttask/PersonDao.java [hunk 24]
```diff
@@ -0,0 +1,234 @@
...
+import java.util.List;
+
+/**
+ * Created by clouway on 16-2-24.
+ */
+public class PersonDao {
+    private String DB_URL;
+    private String user;
+    private String password;
+    private Connection connection;
```
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDao.java]: done

File: src/com/clouway/firsttask/PersonDao.java (hunk 5 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDao.java]: done

File: src/com/clouway/firsttask/PersonDao.java (hunk 6 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDao.java]: done

File: src/com/clouway/firsttask/PersonDao.java (hunk 7 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDao.java]: done

File: src/com/clouway/firsttask/PersonDao.java (hunk 8 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDao.java]: done

File: src/com/clouway/firsttask/PersonDao.java (hunk 9 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDao.java]: done

File: src/com/clouway/firsttask/PersonDao.java (hunk 10 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDao.java]: done

File: src/com/clouway/secondtask/core/Demo.java (hunk 11 above)
Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/core/Demo.java]: done

File: src/com/clouway/secondtask/core/Person.java (hunk 12 above)
Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/core/Person.java]: done

File: src/com/clouway/secondtask/core/PersonDao.java (hunk 13 above)
Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/core/PersonDao.java]: done

File: src/com/clouway/secondtask/core/PersonDao.java (hunk 14 above)
Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/core/PersonDao.java]: done

File: src/com/clouway/secondtask/core/Trip.java (hunk 15 above)
Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/core/Trip.java]: done

File: src/com/clouway/secondtask/core/TripDao.java (hunk 16 above)
Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/core/TripDao.java]: done

File: src/com/clouway/secondtask/core/TripDao.java (hunk 17 above)
Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/core/TripDao.java]: done

File: src/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbcImpl.java (hunk 18 above)
Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbcImpl.java]: done

File: test/com/clouway/firsttask/Tests.java (hunk 19 above)
Comment (kristiyanpetkov) [inline test/com/clouway/firsttask/Tests.java]: done

File: test/com/clouway/firsttask/Tests.java (hunk 20 above)
Comment (kristiyanpetkov) [inline test/com/clouway/firsttask/Tests.java]: done

File: test/com/clouway/firsttask/Tests.java (hunk 21 above)
Comment (kristiyanpetkov) [inline test/com/clouway/firsttask/Tests.java]: done

File: test/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbvImplTest.java (hunk 22 above)
Comment (kristiyanpetkov) [inline test/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbvImplTest.java]: done

File: test/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbvImplTest.java (hunk 23 above)
Comment (kristiyanpetkov) [inline test/com/clouway/secondtask/dataaccesslayer/PeopleTripJdbvImplTest.java]: done

File: src/com/clouway/thirdtask/BackupData.java [hunk 25]
```diff
@@ -0,0 +1,94 @@
+package com.clouway.thirdtask;
+
+import java.sql.*;
+import java.util.Random;
+
+/**
+ * Created by clouway on 16-2-25.
+ */
+public class BackupData {
+    private String DB_URL;
```
Comment (EmilGeorgiev) [inline src/com/clouway/thirdtask/BackupData.java]: .

Comment (kristiyanpetkov) [inline src/com/clouway/thirdtask/BackupData.java]: Fixed. Demo removed. Tests added.

**Comment by mgenov** [comment]
Why this PR is on 9 days ?

File: src/com/clouway/secondtask/core/TripDAO.java [hunk 26]
```diff
@@ -0,0 +1,22 @@
+package com.clouway.secondtask.core;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by clouway on 16-2-24.
+ */
+public interface TripDAO {
```
Comment (mgenov) [inline src/com/clouway/secondtask/core/TripDAO.java]: Unacceptable interface

Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/core/TripDAO.java]: TripDAO interface  renamed to TripRepository.

Comment (mgenov) [inline src/com/clouway/secondtask/core/TripDAO.java]: The problem is not in naming actually.

File: src/com/clouway/firsttask/Person.java [hunk 27]
```diff
@@ -0,0 +1,51 @@
+package com.clouway.firsttask;
+
+/**
+ * Created by clouway on 16-3-1.
+ */
+public class Person {
+    public final String egn;
+    public final String name;
+    public final int age;
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/Person.java]: Use Integer

File: src/com/clouway/firsttask/PersonDAO.java [hunk 28]
```diff
@@ -0,0 +1,222 @@
+package com.clouway.firsttask;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by clouway on 16-2-24.
+ */
+public class PersonDAO {
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDAO.java]: PersistentPersonRepository

File: src/com/clouway/firsttask/PersonDAO.java [hunk 29]
```diff
@@ -0,0 +1,222 @@
...
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    public List<Person> findAll() {
+        Statement stmt = null;
+        ResultSet resultSet = null;
+        List<Person> people = new ArrayList<Person>();
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDAO.java]: add row between ```resultSet``` and ```people```

File: src/com/clouway/firsttask/PersonDAO.java [hunk 30]
```diff
@@ -0,0 +1,222 @@
...
+        try {
+            stmt = connection.createStatement();
+            resultSet = stmt.executeQuery("SELECT * FROM people");
+            while (resultSet.next()) {
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDAO.java]: add row

File: src/com/clouway/firsttask/PersonDAO.java [hunk 31]
```diff
@@ -0,0 +1,222 @@
...
+                String egn = resultSet.getString("egn");
+                String name = resultSet.getString("name");
+                int age = resultSet.getInt("age");
```
Comment (EmilGeorgiev) [inline src/com/clouway/firsttask/PersonDAO.java]: Use Integer

**Comment by mgenov** [comment]
Why title of this PR is meaningless and body is empty ?

File: test/com/clouway/firsttask/Tests.java [hunk 32]
```diff
@@ -0,0 +1,102 @@
...
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * Created by clouway on 16-3-1.
+ */
+public class Tests {
+
+
+    private PersonDAO personDAO;
```
Comment (EmilGeorgiev) [inline test/com/clouway/firsttask/Tests.java]: remove unneccessry rows

File: src/com/clouway/firsttask/Person.java (hunk 27 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/Person.java]: Fixed.

File: src/com/clouway/firsttask/PersonDAO.java (hunk 28 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDAO.java]: Fixed.

File: src/com/clouway/firsttask/PersonDAO.java (hunk 29 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDAO.java]: Fixed.

File: src/com/clouway/firsttask/PersonDAO.java (hunk 30 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDAO.java]: Fixed.

File: src/com/clouway/firsttask/PersonDAO.java (hunk 31 above)
Comment (kristiyanpetkov) [inline src/com/clouway/firsttask/PersonDAO.java]: Fixed.

File: test/com/clouway/firsttask/Tests.java [hunk 33]
```diff
@@ -0,0 +1,102 @@
...
+    @Test
+    public void findAll() {
+        Person myperson = new Person("9109232202", "Kristiyan Petkov", 24, "male");
+        personDAO.register(myperson);
+        Person myperson2 = new Person("9209232201", "Ivan Ivanov", 23, "male");
+        personDAO.register(myperson2);
+        List<Person> got = personDAO.findAll();
+        List<Person> want = new ArrayList<Person>();
+        want.add(myperson);
+        want.add(myperson2);
```
Comment (EmilGeorgiev) [inline test/com/clouway/firsttask/Tests.java]: http://mvnrepository.com/artifact/com.google.guava . With this library you can use LIsts.newArrayList(myperson, myperson2).

File: test/com/clouway/firsttask/Tests.java (hunk 32 above)
Comment (kristiyanpetkov) [inline test/com/clouway/firsttask/Tests.java]: fixed

**Comment by kristiyanpetkov** [comment]
I tried to make the PR title more descriptively.

**Comment by mgenov** [comment]
Do you like it now ?

**Comment by kristiyanpetkov** [comment]
@mgenov, at this moment it is hard to give a good title because i made a pull request with all taks together which is not a good approach. From the next pull request i will separatet each task in a different PR.

File: test/com/clouway/firsttask/Tests.java (hunk 33 above)
Comment (kristiyanpetkov) [inline test/com/clouway/firsttask/Tests.java]: Fixed. Now i use guava lib and the code is less rows and easier to read.

File: src/com/clouway/secondtask/core/PersonRepository.java [hunk 34]
```diff
@@ -0,0 +1,22 @@
...
+ */
+public interface PersonRepository {
+
+    void create(Person person);
+
+    void update(Person person);
+
+    List<Person> findAll();
+
+    List<Person> getAllPeopleNameWhichStartWithSameCharacters(String phrase);
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/core/PersonRepository.java]: findByPartialName()

File: src/com/clouway/secondtask/core/TripRepository.java [hunk 35]
```diff
@@ -0,0 +1,20 @@
+package com.clouway.secondtask.core;
+
+import java.util.List;
+
+/**
+ * Created by clouway on 16-2-24.
+ */
+public interface TripRepository {
+
+    void create(Trip trip);
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/core/TripRepository.java]: it is good practice to add documentations for methods

File: src/com/clouway/secondtask/core/PersonRepository.java [hunk 36]
```diff
@@ -0,0 +1,22 @@
...
+
+    List<Person> findAllPeopleInTheSameCityAtTheSameTime(Date arrival, Date departure);
```
Comment (EmilGeorgiev) [inline src/com/clouway/secondtask/core/PersonRepository.java]: here we can add and parameter city.

File: src/com/clouway/secondtask/core/PersonRepository.java (hunk 34 above)
Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/core/PersonRepository.java]: fixed

**Comment by EmilGeorgiev** [comment]
LGTM

File: src/com/clouway/secondtask/core/PersonRepository.java (hunk 36 above)
Comment (kristiyanpetkov) [inline src/com/clouway/secondtask/core/PersonRepository.java]: fixed

**Pull request merged**